README.md

```diff
@@ -104,14 +106,14 @@ pip install rubicon-ml
 
 ## Develop
 
-`rubicon` uses conda to manage environments. First, install
+`rubicon_ml` uses conda to manage environments. First, install
```

Comment:
```suggestion
`rubicon-ml` uses conda to manage environments. First, install
```

Reply:
I'm not sure about these ones (this and the comment below) because it's wrapped in backticks, but we're about to have them install the package name which uses the hyphen and not the underscore

Reply:
Joe and I agreed on never using backticks on rubicon-ml (pretty arbitrarily) and really only using rubicon-ml for titles and directly referencing installs. does that seem reasonable? 

so based on that I guess we'd want rubicon-ml (no backticks) here, but its definitely one of the weird edge cases

I was planning on using `rubicon_ml` for pretty much everything moving forward